### ISSUE TITLE
[BUG]: XX.5 decimal bug in plan's price value

### ISSUE THREAD

**Issue by Herbak** [getlago/lago#176]
**Describe the bug**
When you create or update a plan, if you precise X.X5 it will cause a bug. For example, 9.94 and 9.96 are correct prices, but 9.95 is blocked by the API.

**To Reproduce**
1. Go to an existing plan and update or create a new plan
2. Go to "Set your plan model" > "Amount" and insert "9.95"
3. An alert "An error occured, please reload the application" pop-up
4. The current GraphQL error appear in the Console : [GraphQL error]: Message: Variable $input of type UpdatePlanInput! was provided invalid value for amountCents (Could not coerce value 994.99 to Int), Path: undefined, Location: [{"line":1,"column":21}]

**Expected behavior**
I expected 9.95 to be a correct price.

**Screenshots**
Not useful but I can provide if necessary

**Support**
 - OS: macOS Ventura
 - Browser : Chrome
 - Version 108.0.5359.124

**Comment by mathieu-lago** [comment]
Hi @Herbak - thanks for reporting this error. We will investigate and get back to you with an update as soon as possible. Sorry for the inconvenience.

**Comment by ansmonjol** [comment]
Hey @Herbak,

I've just merged a fix for this issue. It will be available for you in the next Lago release coming this week :)

Let us know if you have questions.

Closing this issue in the meantime.
